src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -396,19 +400,22 @@ internal ImageView(global::System.IntPtr cPtr, bool cMemoryOwn, bool shown = tru
                 if (_resourceLoadedEventHandler == null)
                 {
                     _resourceLoadedCallback = OnResourceLoaded;
-                    this.ResourceReadySignal(this).Connect(_resourceLoadedCallback);
+                    ViewResourceReadySignal resourceReadySignal = this.ResourceReadySignal(this);
+                    resourceReadySignal?.Connect(_resourceLoadedCallback);
+                    resourceReadySignal?.Dispose();
```

Comment:
this seems to connect native dali signal to nui event. but it do Dispose() for the Signal. 
then does the signal which is connected be destroyed? is this OK?
could you check further?

Reply:
hi, 
I check the code , when new a Signal, the cMemoryOwn is set as false, so I think dispose the variable is Ok, Do I understand that correctly?

Reply:
hi, 
I check the code , when new a Signal, the cMemoryOwn is set as false, so I think dispose the variable is Ok, Do I understand that correctly?